src/util.js

```diff
@@ -43,10 +43,18 @@ var util = {
       return parent;
     });
 
-    child.prototype.init_ = function() {
-      var parents = this.parents || [];
+    child.prototype.init_ = function(a1, a2, a3, a4, a5) {
```

Comment:
@fatihcode would you mind reviewing this addition in particular.

This `init_` function is internal only and is not intended for public API use. However, when making the update to fix the `extends` reserved word usage, I wanted to have children pass arguments to parent constructors.

Rudimentary change, but using `call` with argument lists may not be best case.

Interested in your opinion on this.

Also, happy to revert this update in particular to keep the commit more concise.